doc/sphinxext/flow_diagram.py

```diff
@@ -12,17 +12,20 @@
 sensor_color = "#7bbeca"
 source_color = "#ff6347"
 
-legend = """
-<<FONT POINT-SIZE="%s">
-<TABLE BORDER="0" CELLBORDER="0" CELLSPACING="4" CELLPADDING="4">
-<TR><TD BGCOLOR="%s">    </TD><TD ALIGN="left">
-Sensor (M/EEG) space</TD></TR>
-<TR><TD BGCOLOR="%s">    </TD><TD ALIGN="left">
-Source (brain) space</TD></TR>
-</TABLE></FONT>>""" % (
-    edge_size,
-    sensor_color,
-    source_color,
+legend = (  # noqa: UP031
+    """ # noqa: UP031
```

Comment:
The `# noqa` inside the string is almost certainly an error -- it will cause it to end up in the rendered HTML!